tests/backends/prometheus.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+validate_metrics() {
+	fname="${1}"
+	params="${2}"
+
+	if [ ! -f .gitignore ];	then
+		echo "Need to run as ./tests/backends/$(basename "$0") from top level directory of git repository"
+		return 1
+	fi
+	curl -sS "http://localhost:19999/api/v1/allmetrics?format=prometheus&prefix=nd&timestamps=no${params}" |
+	grep -E 'nd_system_|nd_netdata_|nd_cpu_|nd_services_|nd_system_|nd_net_|nd_disk_|nd_ip_|nd_ipv4_|nd_ipv6_|nd_mem_' |
+	sed -ne 's/{.*//p' | sort | uniq > tests/backends/new-${fname}
+	diff tests/backends/${fname} tests/backends/new-${fname}
+	ret=$?
+	rm tests/backends/new-${fname}
```

Comment:
there is no need to remove files, also in case of an error in `diff` this line won't be executed as everything in bats tests is running with `set -e`

Reply:
Removed the ret